pyproject.toml

```
@@ -0,0 +1,95 @@
+[build-system]
+requires = ["setuptools", "versioningit"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "MembraneCurvature"
```

```suggestion
name = "membrane_curvature"
```
This should do the trick I think